lib/kino/data_table.ex

```diff
@@ -279,17 +295,28 @@ defmodule Kino.DataTable do
     end
   end
 
-  defp value_to_string(value) when is_atom(value), do: inspect(value)
+  defp value_to_string(key, value, nil) do
+    value_to_string(key, value)
+  end
+
+  defp value_to_string(key, value, formatter) do
+    case formatter.(key, value) do
+      {:ok, string} -> string
+      :default -> value_to_string(key, value)
+    end
+  end
+
+  defp value_to_string(_key, value) when is_atom(value), do: inspect(value)
```

Comment:
We can remove the key now, since we never use it in the default logic :D

Reply:
Done!